desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -1794,9 +1794,9 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
       });
 
       self.interpreters = ko.pureComputed(function() {
-        return window.getLastKnownConfig().app_config.editor.interpreters;
+        return window.getLastKnownConfig().app_config.catalogs;
```

Comment:
Might be the same as '.editor.interpreters;' as we need an interpreter to execute the SQL, but probably fine like this